src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc') format('truetype');
```

Comment:
Ensure that the font file 'Avenir.ttc' exists in the 'fonts' directory. Also, verify that the format 'truetype' is correct for this file type. If the font file is not found or the format is incorrect, the custom font will not load properly.